client/state/account-recovery/selectors.js

```diff
@@ -0,0 +1 @@
+export const isFetchingAccountRecoverySettings = ( state ) => state.accountRecovery.isFetchingSettings;
```

Comment:
Thanks for pointing this out. I will make this migration as a separate janitorial PR in the near future :)